README.md

```diff
@@ -1,6 +1,8 @@
 # OpenHMD
 This project aims to provide a Free and Open Source API and drivers for immersive technology, such as head mounted displays with built in head tracking.
 
+Upstream <https://github.com/OpenHMD/OpenHMD>
```

Comment:
This doesn't make sense to be merged upstream.

Reply:
Agreed. It's already been removed in the last commit in this branch (71ccff1). I probably should have cherry picked around it though. Sorry!

Reply:
Let me know if I should do anything else on this one.